elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideConfigProperties.java

```diff
@@ -36,6 +36,11 @@
      */
     private AsyncProperties async;
 
+    /**
+     * Settings for the Dynamic Configuration.
+     */
+    private ControllerProperties dynamicConfig;
```

Comment:
dynamicConfig is not a controller - so why are we using ControllerProperties to configure it?